### DISCUSSION TITLE
怎样在第二页加上指导老师名？

### DISCUSSION THREAD

**Discussion by DBY2-W00** [stone-zeng/fduthesis#260] [Q&A] [answered]
我用的是overleaf上的模板，编译完之后第二页是这样的：
![1](https://user-images.githubusercontent.com/16357305/174560935-c87c4129-b53c-4cc3-840f-e45c32f7906f.png)

但我看到别人的是这样的
![2](https://user-images.githubusercontent.com/16357305/174561009-979f5049-4c8f-4fe0-8260-c30899a82e65.png)

也就是说overleaf上编译出来没有指导老师的名字，怎样修改才能达到下面这张图的效果？

**Comment by note286** [comment]
我猜测不是最新版的原因，你下载GitHub上的版本，再上传到Overleaf上试试。

**Comment by sikouhjw** [comment, marked as answer]
提问之前建议去看下规范是否是如此，因为『你拿来对比的「模板」不一定是正确的』。

根据《复旦大学博士、硕士学位论文规范（2017.03修订版）》[^1]，其中有
> （二）论文撰写格式
> 1、扉页：指导小组成员名单

和
> 一、	内容要求
> 电子版论文应为一个文件，包含如下内容：
> 1、论文封面（参考附件一、标准样本）    
> 2、扉页：指导小组成员名单

可以看出，你拿来对比的『模板』才是错误的，fduthesis 并没有问题。

[^1]: https://gs.fudan.edu.cn/6b/9f/c2806a27551/page.htm
